pom.xml

```diff
@@ -94,7 +94,7 @@
     </developers>
     
     <properties>
-        <caf.worker-framework.version>1.9.0-260</caf.worker-framework.version>
+        <caf.worker-framework.version>3.2.0-SNAPSHOT</caf.worker-framework.version>
```

Comment:
I know this is being wise after the fact but the fact that the major version number was increasing should have informed you that there were breaking changes and triggered you to look at the release notes for both v2 and v3 to see what the breaking changes were.

Reply:
https://github.com/WorkerFramework/worker-framework/releases/tag/v2.0.0
https://github.com/WorkerFramework/worker-framework/releases/tag/v3.0.0